src/Raven.Studio/typescript/viewmodels/database/settings/saveDatabaseSettingsConfirm.ts

```diff
@@ -0,0 +1,32 @@
+import confirmViewModelBase = require("viewmodels/confirmViewModelBase");
+import genUtils = require("common/generalUtils");
+
+class saveDatabaseSettingsConfirm extends confirmViewModelBase<confirmDialogResult> {
+
+    settingsToSaveText: string;
+    
+    constructor(private databaseSettingsToSave: Array<setttingsItem>, private howToReloadDatabaseHtml: string) {
+        super();
+        
+        if (this.databaseSettingsToSave.length === 0) {
```

Comment:
I wonder if we can pass object here and use JSON.stringify + pass through module for colouring?

Reply:
done